### ISSUE TITLE
[Notifications 1.0.0] BUILD FAILURE: Kotlin plugin should be enabled before 'kotlin-android-extensions'

### ISSUE THREAD

**Issue by UsabilitySpace** [cph-cachet/flutter-plugins#179]
### EMULATOR AND OS: 

 - **Emulator**: `Android Studio Emulator` version `Nexus 5X API 28` = `qemu-System-x86_64`
 - **OS**: `iOs` version `10.15.6`

### ISSUE DESCRIPTION: 

After downloading, and when trying to build `flutter-plugins/packages/notifications` project, such a build failure occurs:

```
Kotlin plugin should be enabled before 'kotlin-android-extensions'
``` 

### STEPS TO REPRODUCE: 

1. In Android Studio, choose `File > Open` to open `flutter-plugins/packages/notifications` directory as an existing project.

2. Chose `Run > Edit Configurations...` and then add new Flutter configuration that targets to run `example/lib/main.dart`.

3. Run `Nexus 5X API 28` = `qemu-System-x86_64` Android Studio Emulator.

4. To start building project, press `⌃R` = `Ctrl+R` key combination, or press `Run` button in the Android Studio toolbar: 

![image](https://user-images.githubusercontent.com/46682521/91633266-17a11e00-ea11-11ea-94fb-eed0b6978b2a.png)

### OTHER STEPS TO REPRODUCE: 

The issue persists even if trying the next slightly different steps 1'-3' to reproduce:

1'. In Android Studio, choose `File > Open` to open `flutter-plugins/packages/notifications/example` directory as an existing project (but not `flutter-plugins/packages/notifications/` directory).

2'. Run `Nexus 5X API 28` = `qemu-System-x86_64` Android Studio Emulator.

3'. To start building project, press `⌃R` = `Ctrl+R` key combination, or press `Run` button in the Android Studio toolbar: 

![image](https://user-images.githubusercontent.com/46682521/91633266-17a11e00-ea11-11ea-94fb-eed0b6978b2a.png)

### EXPECTED BEHAVIOUR:

Project should build & run successfully. 

### ACTUAL BEHAVIOUR: 

Project doesn't even build. 

### SCREENSHOTS:

![image](https://user-images.githubusercontent.com/46682521/91633353-cba2a900-ea11-11ea-8a59-ca65a2c6b6f4.png)

### FLUTTER DOCTOR:

```
flutter-plugins/packages/notifications > flutter doctor -v

[✓] Flutter (Channel master, 1.22.0-2.0.pre.36, on Mac OS X 10.15.6 19G73, locale ru-RU)
    • Flutter version 1.22.0-2.0.pre.36 at /Applications/flutter
    • Framework revision d30e36ba8c (7 дней назад), 2020-08-21 22:36:03 -0400
    • Engine revision d495da20d0
    • Dart version 2.10.0 (build 2.10.0-49.0.dev)

 [✓] Android toolchain - develop for Android devices (Android SDK version 30.0.2)
    • Android SDK at /Users/***********/Library/Android/sdk
    • Platform android-30, build-tools 30.0.2
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.6)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.6, Build version 11E708
    • CocoaPods version 1.8.4

[✓] Android Studio (version 4.0)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 48.1.2
    • Dart plugin version 193.7547
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)
 
[✓] Connected device (2 available)            
    • AOSP on IA Emulator (mobile) • emulator-5554 • android-x86 • Android 9 (API 28) (emulator)
    • macOS (desktop)              • macos         • darwin-x64  • Mac OS X 10.15.6 19G73

• No issues found!
```

### THE AIM OF HANDLING THIS ISSUE: 

What are reproducable and exact steps for Android Studio users to overcome this issue in Android Studio under iOS?

**Comment by UsabilitySpace** [comment]
### OTHER STEPS TO REPRODUCE: 

The issue persists even if trying the next slightly different steps 1'-4' to reproduce:

1'. In Android Studio, choose `File > Open` to open `flutter-plugins/packages/notifications/sample` directory as an existing project (but not `flutter-plugins/packages/notifications/` directory).

2'. Run `Nexus 5X API 28` = `qemu-System-x86_64` Android Studio Emulator.

3'. To start building project, press `⌃R` = `Ctrl+R` key combination, or press `Run` button in the Android Studio toolbar: 

![image](https://user-images.githubusercontent.com/46682521/91633266-17a11e00-ea11-11ea-94fb-eed0b6978b2a.png)

**Comment by UsabilitySpace** [comment]
### TO HANDLE THIS ISSUE: 

In `flutter-plugins/packages/notifications/android/app/build.gradle` file replace this two lines: 

`apply plugin: 'kotlin-android-extensions'`
`apply plugin: 'kotlin-android'`

... by this two lines: 

`apply plugin: 'kotlin-android'`
`apply plugin: 'kotlin-android-extensions'`

◼︎

**Comment by thomasnilsson** [comment]
Fixed in version 1.0.1 :)
